Push/PushRegistration.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 import Shared
 import SwiftyJSON
 
-public class PushRegistration: NSObject, NSCoding {
+public class PushRegistration: NSObject, NSSecureCoding {
```

Comment:
we don't sure this `Push` framework, it's Firefoxes, but doesn't hurt to change it